src/components/Button/Button.spec.tsx

```diff
@@ -171,7 +171,7 @@ describe('Components / Button', () => {
       it('should render as button `as={null}`', () => {
         render(
           <ul>
-            <Button as={null as any} label="Something or other" />
+            <Button as={null as null} label="Something or other" />
```

Comment:
@paghar is the type casting here necessary as now you upgraded the `as` prop to `as?: T | null`?

Reply:
@nigellima Should I remove it from the examples?

Reply:
@nigellima 
should I remove example from the documentation?